### ISSUE TITLE
Unified bot mapping

### ISSUE THREAD

**Issue by dave-2** [catapult-project/catapult#4153]
Pinpoint uses three bot mappings:
* bot -> builder (for isolate lookup)
* bot -> browser (for Telemetry)
* bot -> dimensions (for Swarming)
Let's merge them into one mapping!

Also make dashboard `start_try_job.py` use the new mapping.

**Comment by eakuefner** [comment]
Should any of this information be recorded in diagnostics?

**Comment by dave-2** [comment]
Yes, it would be nice to be able to pull the information from the diagnostics.
